.jenkins/pytorch/win-test-helpers/build_pytorch.bat

```diff
@@ -154,6 +154,10 @@ python setup.py install --cmake && sccache --show-stats && (
     copy /Y "build\.ninja_log" "%PYTORCH_FINAL_PACKAGE_DIR%\"
   )
 )
+:: Build functorch on windows
+pushd functorch
+time python setup.py install
+popd
```

Comment:
Windows is a little annoying because we don't transfer docker images between the build job and the test job. What happens is that in the build job, we literally upload the contents of the installed pytorch folder somewhere and then we download the contents during the test job.

Because of how complicated this is I think it's fine if we say this is not going to work on windows. How important is it to get test signal for this on windows?

Reply:
We haven't had a (real) windows-only issue with the functionalization tests so far, so this seems ok to me.